doc/custom_data.rst

```diff
@@ -0,0 +1,138 @@
+
+Example Configuration::
+
+   custom_data:
+     renewables: []
+     elec_demand: false
+     heat_demand: false
+     industry_demand: false
+     industry_database: false
+     transport_demand: false
+     water_costs: false
+     h2_underground: false
+     add_existing: false
+     custom_sectors: false
+     gas_network: false
+     export_ports: false
+     airports: false
+
+Required File Locations and Formats
+-----------------------------------
+
++-------------------------------+----------------------------------------+
+| Custom Data Type              | Required File Path                     |
++===============================+========================================+
+| Gas network                   | ``resources/custom_data/pipelines.csv``|
++-------------------------------+----------------------------------------+
+| Export ports                  | ``data/custom/export_ports.csv``       |
++-------------------------------+----------------------------------------+
+| Airports                      | ``data/custom/airports.csv``           |
++-------------------------------+----------------------------------------+
+| Powerplants                   | ``data/custom_powerplants.csv``        |
++-------------------------------+----------------------------------------+
+| Demand/Renewables Time Series | ``.nc`` files compatible with          |
+|                               | GEGIS/atlite modules                   |
++-------------------------------+----------------------------------------+
+
+Using Linkers
+-------------
+
+Linkers provide a simple interface for integrating custom data:
+
+- Understand and analyze the status of existing data
+- Add new data over simple interfaces
+- Optionally contribute to global databases (e.g., OpenStreetMap, OpenInfraMap)
+
+Currently, linkers support:
+
+- Power lines
+- Custom powerplants
+- Demand time series
+- Renewable production time series
+
+.. note::
+   Additional custom data can be added using **linkers**, which provide
+   simple interfaces for integrating new datasets without modifying core
+   model files.
+
+Reference Data Sources
+----------------------
+
+For guidance on sourcing data, refer to the following table:
+
++------+----------------------------------------+--------+--------+---------+-----+
+| Name | Link                                   | Sector | Global | Country | API |
++======+========================================+========+========+=========+=====+
+| IEA  | https://www.iea.org/countries/         | All    | ✔      |         | ?   |
++------+----------------------------------------+--------+--------+---------+-----+
+| WRI  | https://www.wri.org/                   | All    | ✔      |         | ?   |
++------+----------------------------------------+--------+--------+---------+-----+
+| OECD | https://stats.oecd.org/                | All    | ✔      |         | ?   |
++------+----------------------------------------+--------+--------+---------+-----+
```

Comment:
I am not sure we have a feature like this in the repo. It could be that I also understand. Do you mind explaining what you mean by linkers.